web_touchscreen/static/src/fields.esm.js

```diff
@@ -0,0 +1,27 @@
+import {Field} from "@web/views/fields/field";
+import {hasTouch} from "@web/core/browser/feature_detection";
+import {patch} from "@web/core/utils/patch";
+import {utils} from "@web/core/ui/ui_service";
+
+patch(Field, {
+    /**
+     * @override
+     * Whenever a x2many field loads a list we want to force the kanban view over the
+     * list one if both options are available and a touchscreen is being used.
+     */
+    parseFieldNode(node) {
+        const fieldInfo = super.parseFieldNode(...arguments);
+        // Already handled in super
+        if (utils.isSmall()) {
+            return fieldInfo;
+        }
+        const viewMode = node.getAttribute("mode");
+        const kanban_is_optional =
+            (viewMode && viewMode.split(",").length !== 1) ||
```

Comment:
issue: what if there are other kind of views? I think this is safer:

```suggestion
            (viewMode && viewMode.split(",").includes("kanban")) ||
```

Reply:
All suggestions included :)